cross/redis/Makefile

```diff
@@ -11,11 +11,33 @@ HOMEPAGE = http://redis.io
 COMMENT  = Redis is an open source, advanced key-value store. It is often referred to as a data structure server since keys can contain strings, hashes, lists, sets and sorted sets.
 LICENSE  = BSD
 
+include ../../mk/spksrc.common.mk
+
+
+GNU_CONFIGURE = 1
+
+# Link to atomic library for QorIQ architecture
+ifneq ($(findstring $(ARCH),qoriq),)
+CONFIGURE_TARGET = redis_patch_makefile_to_use_atomic_library
+else
 CONFIGURE_TARGET = nop
-COMPILE_TARGET = redis_custom_compile
+endif
+
+INSTALL_TARGET = redis_custom_install
+
+# use jemalloc for intel arches
+ifeq ($(findstring $(ARCH),$(x64_ARCHES) $(x86_ARCHES)),$(ARCH))
+ENV += USE_JEMALLOC=yes
+else
+ENV += USE_JEMALLOC=no
+endif
```

Comment:
There is `cross/jemalloc` but it is not declared as dependency. I really wonder if that variable has real effect when building/running service

Reply:
the jemalloc source is included in the downloaded redis source archive an built from this.